angular/tech-show-frontend/src/styles.scss

```diff
@@ -0,0 +1,9 @@
+/* You can add global styles to this file, and also import other style files */
+
+* {
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
missing eol
install equalizer if you like

Reply:
I couldn't decide if installing equalizer or keep it as simple as possible. What do you think?

Reply:
I'd use the equalizer due to its low installing complexity